perception/util/visualizer/traffic_light_visualization/src/nodelet.cpp

```diff
@@ -18,43 +18,52 @@
 
 namespace traffic_light
 {
-void TrafficLightRoiVisualizerNodelet::onInit()
-{
-  nh_ = getNodeHandle();
-  pnh_ = getPrivateNodeHandle();
-  image_transport_.reset(new image_transport::ImageTransport(pnh_));
-
-  pnh_.param<bool>("enable_fine_detection", enable_fine_detection_, false);
+TrafficLightRoiVisualizerNodelet::TrafficLightRoiVisualizerNodelet(const rclcpp::NodeOptions & options)
+: Node("traffic_light_roi_visualizer_node", options)
+{  
+  using std::placeholders::_1;
+  using std::placeholders::_2;
+  using std::placeholders::_3;
+  enable_fine_detection_ = this->declare_parameter("enable_fine_detection", false);
```

Comment:
You should be able to do this in the initializer list above.

Reply:
In this particular case we only have 1 parameter, but I think we should be consistent across the code base and since in nodes with more parameters we are doing it within the body of the constructor I think it makes more sense to do the same

Reply:
Fair enough.